services/mbin/mbin.tester.js

```diff
@@ -0,0 +1,63 @@
+import Joi from 'joi'
+import { createServiceTester } from '../tester.js'
+export const t = await createServiceTester()
+
+t.create('get magazine subscribers')
+  .get('/magazine@instance.tld.json')
+  .intercept(nock =>
+    nock('https://instance.tld/')
+      .get('/api/magazine/name/magazine')
+      .reply(
+        200,
+        JSON.stringify({
+          subscriptionsCount: 42,
+        }),
+      ),
+  )
+  .expectBadge({
+    label: 'subscribe to magazine@instance.tld',
+    message: '42',
+    color: 'brightgreen',
+  })
+
+t.create('bad server or connection')
```

Comment:
I don't think this test adds much, it doesn't validate any special behaviour that was implemented in _mbin.service.js_, I'd suggest dropping it. :)

Reply:
Removed